tests/loops/batch/test_truncated_bptt.py

```diff
@@ -170,3 +170,35 @@ def training_step(self, *args, **kwargs):
     )
     trainer.fit(model)
     assert set(trainer.logged_metrics) == {"loss_step", "loss_epoch"}
+
+
+def test_hiddens_multiple_optimizers(tmpdir):
```

Comment:
We were missing a test for the hidden values with multiple optimizers and TBPTT enabled